server/webapp/WEB-INF/rails.new/spec/javascripts/version_updater_spec.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2015 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describe("VersionUpdater", function(){
+    beforeEach(function () {
+        localStorage.clear();
+    });
+
+    describe("update", function(){
+        it("should_update_the_latest_version", function(){
```

Comment:
Using underscores is the old way of writing tests (these were migrated by @jyotisingh, AFAIK). Please use sentences.